application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarEventSheet.xml

```diff
@@ -256,6 +293,21 @@ $doc.display($field)
     : $doc.display('allDay')
     ; &lt;label for="description"&gt;$escapetool.xml($doc.displayPrettyName('description', false, false))&lt;/label&gt;
     : $eventInstance.getDescriptionHtml()
+      ## The colors here are for the XWiki Object (from the page url).
+      ## Colors displayed in the calendar are defined in the MoccaCalendarScriptService.java file.
+      ## There is another validation check made in MoccaCalendarScriptService.java. See the fillInColorsFromNearestCalendar() method.
```

Comment:
This comment is not very helpful. Some things related to understanding the code should be left to be discovered by just looking at the code when it's needed, in order to not make the code harder to read. For this example specific I feel like it would be also hard to follow, since in case the code changes, this comment will remain outdated, doing more harm than good
You should add a comment if you feel some things are harder to be spotted, or something complex is involved